docs/fr/developers/03_Backend/05_Extensions.md

```diff
@@ -332,4 +332,4 @@ TODO :
 
 ### Écrire le fichier configure.phtml
 
-TODO
+Lorsque vous voulez soutenir les configurations utilisateur ou simplement afficher quelques informations, vous devez créer le fichier `configure.phtml`.
```

Comment:
@Alkarex The English was lying by saying something should be translated from French when it was the reverse. This is my translation for the following, but my French isn't stellar:

> When you want to support user configurations for your extension or simply display some information, you have to create the `configure.phtml` file.

Reply:
@Frenzie The following is better:

> Lorsque vous voulez ajouter de la configuration à votre extension ou afficher ses informations, vous devez créer le fichier `configure.phtml`

Reply:
@aledeg Thanks! Btw feel free to comment on the English too; I'll give it another read in a couple of days but it's basically done.

Reply:
@Frenzie Sure, but at the moment, it's pretty crazy and I don't have much time to spend on FRSS. It's sad but I am trying to keep an eye on it.